src/ide/spans.ts

```diff
@@ -617,7 +617,7 @@ export class CodeBlockSpan extends BlockSpan {
   protected footerWidgetElem:HTMLElement;
 
   apply(from:Position, to:Position, origin = "+input") {
-    this.lineBackgroundClass = "code";
+    this.lineBackgroundClass = "code" + ((this.source.info && this.source.info.toLowerCase().indexOf("css") === 0) ? " css" : "");
```

Comment:
There's an interesting question here of whether or not blocks of other languages may be disabled. Since CSS has no stateful side effects, it seems like a good candidate for supporting that.

Either way, it seems like it might be nice to abstract the info parsing logic into it's own set of functions, instead of doing it ad-hoc on a per-syntax basis. a `syntax():string` method returning the syntax portion of the info if present would be a good start.